Packs/Cyberint/Integrations/FeedCyberint/FeedCyberint.yml

```
@@ -198,6 +198,114 @@ script:
       description: Description.
       type: String
     description: Gets indicators from the feed.
+  - name: cyberint-get-file-sha256
+    arguments:
+    - name: value
+      description: SHA256 string.
+    outputs:
+    - contextPath: Cyberint.detected_activity
+      description: Detected activity.
+      type: String
+    - contextPath: Cyberint.ioc_type
+      description: The indicator type.
+      type: String
+    - contextPath: Cyberint.ioc_value
+      description: The indicator value.
+      type: String
+    - contextPath: Cyberint.observation_date
+      description: Observation date.
+      type: String
+    - contextPath: Cyberint.severity_score
+      description: Severity score.
+      type: String
+    - contextPath: Cyberint.confidence
+      description: Confidence.
+      type: String
+    - contextPath: Cyberint.description
+      description: Description.
+      type: String
+    description: Gets File SHA256 enrichment from the feed.
+  - name: cyberint-get-domain
+    arguments:
+    - name: value
+      description: Domain string.
+    outputs:
+    - contextPath: Cyberint.detected_activity
```

Please fix all the contextPath according to the update of the outputs_prefix, like this:
```suggestion
    - contextPath: Cyberint.domain.detected_activity
```

---

Fixed